analyzer.go

```diff
@@ -0,0 +1,206 @@
+package wraperr
+
+import (
+	"bufio"
+	"go/ast"
+	"go/token"
+	"go/types"
+	"os"
+	"strings"
+
+	"golang.org/x/tools/go/analysis"
+	"golang.org/x/tools/go/analysis/passes/inspect"
+	"golang.org/x/tools/go/ast/inspector"
+)
+
+var Analyzer = &analysis.Analyzer{
+	Name:     "wraperr",
+	Doc:      "Check that error return value are wrapped",
+	Requires: []*analysis.Analyzer{inspect.Analyzer},
```

Comment:
**[errcheck]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
 cannot use inspect.Analyzer (variable of type *invalid type) as *golang.org/x/tools/go/analysis.Analyzer value in array or slice literal